packages/contracts-bedrock/src/L1/L2OutputOracle.sol

```diff
@@ -79,19 +83,17 @@ contract L2OutputOracle is Initializable, ISemver {
         L2_BLOCK_TIME = _l2BlockTime;
         FINALIZATION_PERIOD_SECONDS = _finalizationPeriodSeconds;
 
-        initialize({ _startingBlockNumber: 0, _startingTimestamp: 0, _proposer: address(0), _challenger: address(0) });
+        initialize({ _startingBlockNumber: 0, _startingTimestamp: 0, _systemConfig: SystemConfig(address(0)) });
     }
 
     /// @notice Initializer.
     /// @param _startingBlockNumber Block number for the first recoded L2 block.
     /// @param _startingTimestamp   Timestamp for the first recoded L2 block.
-    /// @param _proposer            The address of the proposer.
-    /// @param _challenger          The address of the challenger.
+    /// @param _systemConfig        The address of the system config.
     function initialize(
         uint256 _startingBlockNumber,
         uint256 _startingTimestamp,
-        address _proposer,
-        address _challenger
+        SystemConfig _systemConfig
     )
         public
         reinitializer(2)
```

Comment:
We will want to make this `reinitializer(3)` at some point in the future but its fine for now

Reply:
Hmmm, is there any reason _not_ to make it `3` now?